assets/css/first-section.css

```diff
@@ -0,0 +1,23 @@
+.first-section {
+  background-color: var(--first-light-background);
+}
+
+.first-section-title {
+  text-align: center;
+  padding-top: 24px;
+  font-weight: var(--bold-font);
+}
+
+.first-section-container {
+  padding-top: 24px;
```

Comment:
Please use spacing variable for all spaces. See [this link](https://developer.mozilla.org/en-US/docs/Web/CSS/calc()). Will help you :D

Reply:
Done.